dist/source/repo-creator/sponsorship.ts

```diff
@@ -0,0 +1,64 @@
+import { RepoCreator } from 'source/services/RepoCreator';
+import { Repository as RepositoryWireModel } from 'source/models/Repository';
+import { OAuth } from 'source/services/OAuth-Auth0';
+import { GitHub } from 'source/services/GitHub';
+import { StripeCheckout, StripeToken } from 'source/services/StripeCheckout';
```

Comment:
Once the `stripeCheckout` is removed from the constructor this import won't be necessary.